functional_tests/scylla_operator/test_functional.py

```diff
@@ -332,3 +332,21 @@ def test_scylla_operator_pods(db_cluster: ScyllaPodCluster):
 
     not_running_pods = ','.join([pods_info[0] for pods_info in scylla_operator_pods if pods_info[1] != 'Running'])
     assert not not_running_pods, f'There are pods in state other than running: {not_running_pods}'
+
+
+def test_startup_probe_in_scylla_pods(db_cluster: ScyllaPodCluster):
+    pods = db_cluster.k8s_cluster.kubectl('get pods -A -l "app.kubernetes.io/name=scylla" -o yaml')
+
+    no_startup_probe_pods = []
```

Comment:
Great idea, but let's make it even better
```suggestion
    no_startup_probe_pods = []
    for pod in yaml.load(pods.stdout)["items"]:
        for container in pod.get("spec", {}).get("containers", []):
            if container['name'] == "scylla" and not container.get("startupProbe"):
                no_startup_probe_pods.append(f"pod: {pod['metadata']['name']}, container: {container['name']}")
```

Reply:
done